### ISSUE TITLE
popup_closed_by_user

### ISSUE THREAD

**Issue by vov4ik08** [sabyasachibiswal/angular5-social-login#4]
Hello. I try to use google auth with your library. After config by your readme and click to login button by google i can seed google window with my account. After select account in console display error 'popup_closed_by_user' http://prntscr.com/ijzh8r. Please help me

**Comment by revinfotech2** [comment]
Same here.I am getting this response only in case of google.

**Comment by akshayseth** [comment]
Same here.I am getting this error only in the case of google.Please help.

**Comment by marcoantoniocanhas** [comment]
Same here. 

`"angular5-social-login": "1.0.9"`

**Comment by manumohant** [comment]
same here......

**Comment by Shakell** [comment]
same here......

**Comment by maxtsaryk** [comment]
the same

**Comment by manishbt** [comment]
Same issue

**Comment by piyush-ampersand** [comment]
Same here , has someone found a solution for this ??

**Comment by manishbt** [comment]
yes, i have working solution for this issue, wait i will share the link which is working perfectly.
https://github.com/KbaylonH/angularx-social-login/tree/83f977f65c63d11c1399d553b6e84a768a4b0ab3

use this module in your application, it works perfectly

**Comment by nkonsta** [comment]
I was able to solve this problem by explicitly define where this plugin should be allowed to work in google console developer. Under credentials find the OAuth 2.0 client IDs section and add the address from which you will access the plugin. And for those having an error of undefined object when calling the signin method of this plugin in chrome and firefox, I was able to solve this as well by changing https://apis.google.com/js/platform.js into https://apis.google.com/js/api.js under node_modules/angular5-social-login/providers/angular5-social-login.umd.js

**Comment by orfeapapa** [comment]
We see these error in chrome ecognito.
